src/org/rust/lang/RustSdkType.kt

```diff
@@ -67,10 +85,22 @@ class RustSdkType : SdkType("Rust SDK") {
         }
 
         try {
-            val cmd = GeneralCommandLine()
+            val cmd = if (isMultiRust(sdkHome)) {
+                val sdkLibraryPath = getSdkLibrary(sdkHome).absolutePath
+                GeneralCommandLine()
+                    .withWorkDirectory(sdkHome)
+                    .withEnvironment(hashMapOf(
+                        "DYLD_LIBRARY_PATH" to sdkLibraryPath,
+                        "LD_LIBRARY_PATH" to sdkLibraryPath
+                    ))
+                    .withExePath(rustc.absolutePath)
+                    .withParameters("-C rpath", "--version")
```

Comment:
I just ported the solution from the ```/usr/local/bin/rustc``` script placed there by multirust. Don't know how to improve/reuse it. Maybe @atsky could help.